nautobot/extras/tests/test_api.py

```diff
@@ -755,6 +756,52 @@ def setUpTestData(cls):
         """
 
 
+class GraphQLQueryTest(APIViewTestCases.APIViewTestCase):
+    model = GraphQLQuery
+    brief_fields = ["display", "id", "name", "url"]
+
+    create_data = [
+        {
+            "name": "graphql-query-4",
+            "slug": "graphql-query-4",
+            "query": "{ query: sites {name} }",
+        },
+        {
+            "name": "graphql-query-5",
+            "slug": "graphql-query-5",
+            "query": '{ devices(role: "edge") { id, name, device_role { name slug } } }',
+        },
+        {
+            "name": "graphql-query-6",
+            "slug": "graphql-query-6",
+            "query": "query ($device: String!) { devices(name: $device) { config_context name position serial primary_ip4 { id primary_ip4_for { id name } } tenant { name } tags { name slug } device_role { name } platform { name slug manufacturer { name } napalm_driver } site { name slug vlans { id name vid } vlan_groups { id } } interfaces { description mac_address enabled name ip_addresses { address tags { id } } connected_circuit_termination { circuit { cid commit_rate provider { name } } } tagged_vlans { id } untagged_vlan { id } cable { termination_a_type status { name } color } tagged_vlans { site { name } id } tags { id } } } }",
```

Comment:
```suggestion
            "query": "query ($device: String!) { devices(name: $device) { config_context name position serial primary_ip4 "
            "{ id primary_ip4_for { id name } } tenant { name } tags { name slug } device_role { name } platform "
            "{ name slug manufacturer { name } napalm_driver } site { name slug vlans { id name vid } vlan_groups "
            "{ id } } interfaces { description mac_address enabled name ip_addresses { address tags { id } } "
            "connected_circuit_termination { circuit { cid commit_rate provider { name } } } tagged_vlans { id } "
            "untagged_vlan { id } cable { termination_a_type status { name } color } tagged_vlans { site { name } id } "
            "tags { id } } } }",
```

Similar recommendation for long query strings below... alternately you could pretty-print them and use triple-quotes, a la:

```suggestion
"query": """\
query ($device: String!) {
  devices(name: $device) {
    config_context
    name
    position
    serial
    primary_ip4 {
      id
      primary_ip4_for {
        id
        name
      }
    }
    tenant {
      name
    }
    tags {
      name
      slug
    }
    device_role {
      name
    }
    platform {
      name
      slug
      manufacturer {
        name
      }
      napalm_driver
    }
    site {
      name
      slug 
      vlans {
        id
        name
        vid
      }
      vlan_groups {
        id
      }
    }
    interfaces {
      description
      mac_address
      enabled
      name
      ip_addresses {
        address
        tags {
          id
        }
      }
      connected_circuit_termination {
        circuit {
          cid
          commit_rate
          provider {
            name
          }
        }
      }
      tagged_vlans {
        id
      }
      untagged_vlan {
        id
      }
      cable {
        termination_a_type
        status {
          name
        }
        color
      }
      tagged_vlans {
        site {
          name
        }
        id
      }
      tags {
        id
      }
    }
  }
}""",
```